legal-api/flags.json

```diff
@@ -4,6 +4,24 @@
         "bool-flag": true,
         "integer-flag": 10,
         "enable-legal-name-fix": true,
-        "disable-nr-check": false
+        "disable-nr-check": true,
+        "db-versioning": {
+            "initialize-old-versioning": true,
+            "initialize-new-versioning": false,
+            "enable-new-versioning": {
+                "legal-api": true,
+                "legal-api-versioning-class": true,
+                "emailer": false,
+                "filer": false,
+                "entity-bn": false,
+                "digital-credentials": false,
+                "dissolutions-job": false,
+                "furnishings-job": false,
+                "emailer-reminder-job": false,
+                "future-effective-job": false,
+                "update-colin-filings-job": false,
+                "update-legal-filings-job": false
+            }
+        }
     }
```

Comment:
feature flag used to toggle old & new db versioning.  in the case where new db versioning is enabled, to support incremental updates, each BE component can be turned on when new db versioning is integrated into component and ready to go.